### PR TITLE
Moving timescaledb to pkg instead of building

### PR DESCRIPTION
**Pull request by idanovinda**
decrease build time by moving timescaledb to pkg instead of building

### CODE REVIEW

File: postgres-appliance/build_scripts/base.sh [hunk 1]
```diff
@@ -96,7 +100,8 @@ for version in $DEB_PG_SUPPORTED_VERSIONS; do
                 "postgresql-${version}-repack"
                 "postgresql-${version}-wal2json"
                 "postgresql-${version}-pllua"
-                "postgresql-${version}-pgvector")
+                "postgresql-${version}-pgvector"
+                "timescaledb-2-postgresql-${version}")
```
Comment (hughcapet) [inline postgres-appliance/build_scripts/base.sh:104]: pkg name is different depending on the `$TIMESCALEDB_APACHE_ONLY` value set

**Comment by hughcapet** [comment]
when building with ` --build-arg TIMESCALEDB_TOOLKIT=true --build-arg TIMESCALEDB_APACHE_ONLY=false`:

```
+ rm /etc/apt/sources.list.d/timescaledb.list
+ rm /usr/share/keyrings/timescale_E7391C94080429FF.gpg
rm: cannot remove '/usr/share/keyrings/timescale_E7391C94080429FF.gpg': No such file or directory
```

File: postgres-appliance/build_scripts/base.sh [hunk 2]
```diff
@@ -69,13 +69,14 @@ apt-get install -y \
...
-# add TimescaleDB repository
-DISTRIB_CODENAME=$(sed </etc/os-release -ne 's/^VERSION_CODENAME=//p')
-echo "deb [signed-by=/etc/apt/keyrings/timescale_timescaledb-archive-keyring.gpg] https://packagecloud.io/timescale/timescaledb/ubuntu/ ${DISTRIB_CODENAME} main" | tee /etc/apt/sources.list.d/timescaledb.list
-curl -fsSL https://packagecloud.io/timescale/timescaledb/gpgkey | gpg --dearmor | tee /etc/apt/keyrings/timescale_timescaledb-archive-keyring.gpg > /dev/null
-
 for version in $DEB_PG_SUPPORTED_VERSIONS; do
     sed -i "s/ main.*$/ main $version/g" /etc/apt/sources.list.d/pgdg.list
+
+    # add TimescaleDB repository
+    DISTRIB_CODENAME=$(sed </etc/os-release -ne 's/^VERSION_CODENAME=//p')
```
Comment (hughcapet) [inline postgres-appliance/build_scripts/base.sh:76]: i would better move this to the prepare step
https://github.com/zalando/spilo/blob/9edde4d2929d25b27fb50663493ddb056854f4fc/postgres-appliance/build_scripts/prepare.sh#L36-L41
and likely deletion is not needed at all

**Review by hughcapet — APPROVED** [review]

**Comment by idanovinda** [comment]
👍

**Pull request merged**